### PR TITLE
UI and translation bug fixes

### PR DESCRIPTION
**Pull request by abbodh**
Translation changes
- User facing resource string changes at client app and strings.resx file.

UI bug fixes includes
- Fixed long message title truncate issue in comms tab.
- List header alignment issue in manage group tab for mobile screens.
- Vertical scroll overlapping in discover groups tab.
- Image height fixes in Teams group tab.

### CODE REVIEW

File: Source/DIConnect/ClientApp/src/components/Messages/messages.tsx [hunk 1]
```diff
@@ -18,252 +18,252 @@ import { TFunction } from "i18next";
...
+                                <ExclamationCircleIcon xSpacing="both" className="unknown" outline />
+                                <span className="semiBold">{formatNumber(message.unknown)}</span>
+                            </TooltipHost>
+                        }
+                    </div>
+                </Flex.Item>
+                <Flex.Item size="size.quarter" variables={{ 'size.quarter': '24%' }} >
+                    <Text
+                        truncated
+                        className="semiBold"
```
Comment (aosolis) [inline Source/DIConnect/ClientApp/src/components/Messages/messages.tsx:228]: Doesn’t Text support weight=semibold?

Comment (v-haalam) [reply Source/DIConnect/ClientApp/src/components/Messages/messages.tsx:228]: Text supports weight="semibold", however here using custom css for text across page

File: Source/DIConnect/ClientApp/public/locales/en-US/translation.json [hunk 2]
```diff
@@ -42,15 +42,15 @@
   "SendToRosters": "Send in chat to members of the following teams",
   "SentToRosters": "Sent in chat to people in teams",
   "SendToRostersPlaceHolder": "Choose team(s) members",
-  "SendToGroups": "Send in chat to members of the following M365 groups, Distribution groups or Security groups",
+  "SendToGroups": "Send in chat to members of the group(s) you specify (E.g., M365 group, Distribution group, or Security group.)",
```
Comment (aosolis) [inline Source/DIConnect/ClientApp/public/locales/en-US/translation.json:45]: e.g.
(all lower case)
Better yet, "for example"
Also no period at the end of "Security group"

File: Source/DIConnect/ClientApp/src/components/Messages/messages.tsx (hunk 1 above)
Comment (aosolis) [reply Source/DIConnect/ClientApp/src/components/Messages/messages.tsx:228]: Why, though?

**Review by aosolis — APPROVED** [review]

File: Source/DIConnect/ClientApp/public/locales/en-US/translation.json (hunk 2 above)
Comment (abbodh) [reply Source/DIConnect/ClientApp/public/locales/en-US/translation.json:45]: updated

File: Source/DIConnect/ClientApp/src/components/Messages/messages.tsx (hunk 1 above)
Comment (abbodh) [reply Source/DIConnect/ClientApp/src/components/Messages/messages.tsx:228]: removed it for Text element only. semiBold class is still used by few spans in the same file, so haven't removed it from scss.

**Pull request merged**